src/app/components/stacks-transaction-item/transaction-action-button.tsx

```diff
@@ -1,37 +1,42 @@
 import { HStack, styled } from 'leather-styles/jsx';
 
-import { ChevronsRightIcon } from '@leather.io/ui';
-
-interface IncreaseFeeButtonProps {
+interface ActionButtonProps {
+  icon?: React.ReactNode;
   isEnabled?: boolean;
   isSelected: boolean;
-  onIncreaseFee(): void;
+  label: string;
+  maxWidth?: string;
+  onButtonClick(): void;
+  textColor?: string;
 }
-export function IncreaseFeeButton(props: IncreaseFeeButtonProps) {
-  const { isEnabled, isSelected, onIncreaseFee } = props;
+
+export function TransactionActionButton(props: ActionButtonProps) {
+  const { isEnabled, isSelected, onButtonClick, label, icon, textColor, maxWidth } = props;
   const isActive = isEnabled && !isSelected;
 
+  if (!isActive) return null;
+
   return (
     <styled.button
       _hover={{ color: 'ink.text-subdued' }}
       bg="ink.background-primary"
-      maxWidth="110px"
+      maxWidth={maxWidth}
```

Comment:
Maybe you can just keep the `maxWidth="110px"` rather than exposing it as a prop and passing it in ?

Reply:
> Maybe you can just keep the `maxWidth="110px"` rather than exposing it as a prop and passing it in ?

@pete-watters  yeah did this too in my PR; i was trying to implement the integration test but looks like there are a lot of selectors to be added. Can i leave the integration test to the team? i can fix the newly added reviews and also rebase with new changes.